test-playwright/VvFormField.v4.spec.ts

```
@@ -1,32 +1,29 @@
-import { test, expect } from '@playwright/experimental-ct-vue'
+import { test, expect } from 'vitest'
```

ESLint

[error] Prefer using it instead of test

(test/consistent-test-it)